### ISSUE TITLE
Typo Chapter 1 

### ISSUE THREAD

**Issue by JayKan** [vkarpov15/generators-book-issues#1]
From Chapter 1, page 8, under "Iterables and Generators" section where you have explained the difference between generator objects vs. generator functions. In your example, you have,

"fibonacciGenerator[Symbol.iterable]; // Undefined
fibonacciGenerator(10)[Symbol.iterable]; // Function"

shouldn't those two lines be: 

"fibonacciGenerator[Symbol. iterator]; // Undefined
fibonacciGenerator(10)[Symbol. iterator]; // Function
"

In short, you have used [Symbol.iterable] instead of [Symbol.iterator] in your example code.

Thanks,
Jay Kan

**Comment by vkarpov15** [comment]
You're right, that's a typo on my part. Fixed now. Let me know if you want an updated copy.

**Comment by JayKan** [comment]
@vkarpov15,

It's all good. Your tutorials were great. Any other good resources/examples in regards to `generators` that you would recommend besides `koa` and `Co`?

Thanks,
Jay Kan

**Comment by vkarpov15** [comment]
I've written a few articles about it on my blog: http://thecodebarbarian.com/tag/nodejs.html . http://www.2ality.com/2015/03/es6-generators.html is also a great read.

**Comment by vkarpov15** [comment]
Also TJ wrote a pretty solid article: https://medium.com/@tjholowaychuk/callbacks-vs-coroutines-174f1fe66127

**Comment by JayKan** [comment]
Thanks a lot @vkarpov15!
